src/sentry/incidents/serializers/alert_rule_trigger_action.py

```diff
@@ -99,7 +103,8 @@ def validate(self, attrs):
                     }
                 )
 
-        if attrs.get("type") == AlertRuleTriggerAction.Type.EMAIL:
+        action_type = attrs.get("type")
+        if action_type == AlertRuleTriggerAction.Type.EMAIL:
```

Comment:
a good candidate to user [match/case](https://docs.python.org/3/whatsnew/3.10.html#pep-634-structural-pattern-matching) instead ;-)